catalogs/catalog-fileset/src/main/java/com/datastrato/gravitino/catalog/file/HadoopCatalogOperations.java

```diff
@@ -150,4 +340,44 @@ public PropertiesMetadata filesetPropertiesMetadata() throws UnsupportedOperatio
 
   @Override
   public void close() throws IOException {}
+
+  private Map<String, String> addManagedFlagToProperties(Map<String, String> properties) {
+    return ImmutableMap.<String, String>builder()
+        .putAll(properties)
+        .put(HadoopSchemaPropertiesMetadata.GRAVITINO_MANAGED_ENTITY, Boolean.TRUE.toString())
```

Comment:
`HadoopSchemaPropertiesMetadata.GRAVITINO_MANAGED_ENTITY` should be a reserved property by gravitino. should it be protected from being modified by `AlterSchema`?

Reply:
Yes, this is a reserved property, it is invisible and unmodified to users, it will throw an exception when trying to modify it. You can see `PropertiesMetadata#validatePropertyForAlter`.